torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -753,6 +829,13 @@ def _run_param_init_fn():
             for p in self.params:
                 self._offload_to_cpu(p)
 
+        if needs_move_back_to_cpu:
+            # _fsdp_wrapped_module took ownership of module and we deleted
+            # module, so move _fsdp_wrapped_module back.
+            self._fsdp_wrapped_module = self._fsdp_wrapped_module.to(
+                torch.device("cpu")
+            )
+
```

Comment:
just want to make sure this is a blocking operation, right?

Reply:
yep, `non_blocking=False` is the default